MANIFEST.in

```diff
@@ -1,8 +1,6 @@
 include LICENSE
-include README.md
```

Comment:
this gets included by default (docs [here](https://packaging.python.org/en/latest/guides/using-manifest-in/))